inspirehep/dojson/hep/fields/bd5xx.py

```diff
@@ -235,12 +235,19 @@ def license2marc(self, key, value):
 @hep.over('acquisition_source', '^541[10_].')
 def acquisition_source(self, key, value):
     """Immediate Source of Acquisition Note."""
+    def _get_source(value):
+        sources = force_force_list(value.get('a'))
+        sources_without_inspire = [
+            el for el in sources if not el.startswith('inspire')]
```

Comment:
can you really specify `inspire:uid:` prefix, so that we are sure not to throw away something else that is still sensible?